src/main/java/org/fundacionjala/sfdc/commons/PropertiesManager.java

```diff
@@ -0,0 +1,55 @@
+package org.fundacionjala.sfdc.commons;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * PropertiesManager.java.
+ * Class that applies Singleton pattern to instance Properties only once.
+ */
+public final class PropertiesManager {
+    private static PropertiesManager propertiesManager;
+    private Properties prop;
+    private InputStream input = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "input" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)